codec.proto

```diff
@@ -40,11 +40,6 @@ message EventData {
   EventBlock block = 2;
 }
 
-message TransactionData {
```

Comment:
This is needed for transaction handler, it shouldn't be removed.

Reply:
I need to restore it